src/shogun/optimization/ElasticNetPenalty.h

```diff
@@ -92,11 +102,20 @@ class ElasticNetPenalty: public SparsePenalty
 		return grad;
 	}
 
-	virtual void set_rounding_eplison(float64_t eplison)
+	/** Set the rounding epslion for L1 penalty
+	 *
+	 * @param epslion rounding epslion
+	 *
+	 */
+	virtual void set_rounding_epslion(float64_t epslion)
 	{
-		m_l1_penalty->set_rounding_eplison(eplison);
+		m_l1_penalty->set_rounding_epslion(epslion);
 	}
 
+	/** Do proximal projection/operation in place
```

Comment:
New line to be consistent between this line and the next. There are similar cases to this case and the one above in other methods in this file and files below.